models/zuora/zuora_subscription.sql

```diff
@@ -1,12 +1,158 @@
+with sub_mrr as (
 select
-    id                                  as subscr_id,
-    status                              as subscr_status,
-    termtype                            as subscr_term_type,
-    accountid                           as account_id,
-    contracteffectivedate::timestamp    as subscr_start,
-    subscriptionenddate::timestamp      as subscr_end,
-    name                                as subscr_name,
-    "version#392c30e6081c24fb78ddf6d622de4f33"::integer 
-                                        as subscr_version,
-    *
-from zuora.zuora_subscription
+  c.id as sub_id,
+  sum(cast(a.mrr as decimal(18,2))) as mrr,
+  sum(cast(a.tcv as decimal(18,2))) as tcv
+from zuora.zuora_rate_plan_charge a
```

Comment:
@dwallacerjm Once a base model is created, we should not reference the original table. This should be `{{env.schema}}.zuora_rate_plan_charge`, where the `{{env.schema}}` indicates that we are using a base model.